app/api/auth/login/individual/route.ts

```diff
@@ -0,0 +1,45 @@
+import { ConflictError } from "@/custom/errors/dictionary/errorDictionary";
+import { handleErrorEdgeCases } from "@/custom/errors/handler/errorHandler";
+import { connectMongoDB } from "@/lib/mongo_connect/mongoConnect";
+import {
+  AccountIndividual,
+  type IndividualSchemaTypes,
+} from "@/models/auth/IndividualSchema";
+import bcrypt from "bcrypt";
+import { NextResponse as res } from "next/server";
+
+export async function POST(request: Request) {
+  try {
+    await connectMongoDB();
+
+    const data = await request.json();
+
+    const { email, password } = data;
+
+    const user = await AccountIndividual.findOne<IndividualSchemaTypes>(
+      {
+        email,
+      },
+      {
+        user_id: 1,
+        password: 1,
+      }
+    );
+
+    if (!user) throw new ConflictError("Invalid credentials");
+
+    const isPasswordMatch = bcrypt.compareSync(password, user.password);
+
+    if (!isPasswordMatch) throw new ConflictError("Invalid credentials");
+
+    const { user_id } = user;
+
+    return res.json({
+      status: 201,
+      message: "Login successfull",
+      id: user_id,
+    });
+  } catch (error) {
+    return handleErrorEdgeCases(error);
```

Comment:
The method for handling errors here has been updated. Please refer to the signup routes for reference.

The updates will be available after the next commit on my end

Reply:
yes, i saw that